content/issue-2/optimize-rust-comptime.md

```diff
@@ -0,0 +1,200 @@
+
+Although there are often complaints saying Rust'compilation speed is notorious slow, our project [RisingWave](https://github.com/risingwavelabs/risingwave) is not very slow to compile, especially since previously contributors like ([skyzh](https://github.com/skyzh), [BugenZhao](https://github.com/bugenzhao)) have put in a lot of effort. After using an M1 MacBook Pro, compiling is not a problem at all. A full debug compilation only takes 2-3 minutes.
```

Comment:
Just a tiny couple of things - Can we change notorious to notoriously ?
And "is not very slow to compile" to "is not so slow to compile"

Reply:
Do I need to revert it?

Reply:
Let me fix it.